store.html

```diff
@@ -0,0 +1,1097 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>Karma Store</title>
+    <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
+    <script src="https://kit.fontawesome.com/a6d80728cd.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+      <header class="store_header">
+                <img class="header_logo" id="logo"
+        src="img/karma-logo.svg"
+        alt="Karma logo"
+          />
+      <nav>
+        <a href="/index.html">Meet Karma</a>
+        <a href="">How it Works</a>
+        <a href="">Store</a>
+        <a href="">Blog</a>
+        <a href="">Help</a>
+        <a href="">Login</a>
+      </nav>
+    </header>
+
+  
+<main>
+<section class="store_order">
+
+  <div>
+      <form action="/store_page" class="store-form">
+    
+        <div>
+  <h1 class="storehdr">Order your Karma Wifi device today!</h1>
+</div>
+          <div style="float:left;" class="item">
```

Comment:
Using `float` is not generally recommended.

I am not too sure why as I'm not very familiar with CSS myself. 
I found an interesting discussion in stackoverflow https://stackoverflow.com/questions/9776840/is-float-for-layout-bad-what-should-be-used-in-its-place